ext/standard/tests/network/getprotobyname_basic.phpt

```diff
@@ -2,6 +2,12 @@
 getprotobyname function basic test
 --CREDITS--
 edgarsandi - <edgar.r.sandi@gmail.com>
+--SKIPIF--
+<?php
+    if(in_array(PHP_OS_FAMILY, ['BSD', 'Darwin', 'Solaris', 'Linux'])){
```

Comment:
couldn't you just do the feature detect without this OS whitelist?

Reply:
I've added the OS whitelist to prevent the test from being skipped on Windows and others (`PHP_OS_FAMILY=Unknown`). As far as I know only *nix has the `/etc/protocols` file which is read by C `getprotobyname` / `getprotobynumber`.